Readme.md

```diff
@@ -0,0 +1,60 @@
+**Pre-requisite:**
+
+1. python2
+2. gcloud
+
+Follow these set-up before running the benchmarking script.
+
+1. Keep your envoy-binary in an accessible location, `$ENVOY_BINARY`.
+2. Keep all the scripts, Makefiles in a directory, `$SCRIPTS`. The scripts, Makefiles refer to the same files that are included in this benchmarking package.
+3. Keep all the Envoy configurations in a directory, `$ENVOY_CONFIG`.
+4. Select a directory in which you want to keep the result, `$RESULT`.
+
+Install the following packages (possibly running the below commands): `pexpect`
+
+1. `sudo pip install pexpect`
```

Comment:
And I'd consider this a pre-req. Just add it above, and consider adding commands such as that you've included here for installing the other items.